docs/products/postgresql/howto/pgbouncer-stats.rst

```diff
@@ -0,0 +1,37 @@
+Accessing PGBouncer statistics
+==============================
+
+PGBouncer is used at Aiven for :doc:`../concepts/pg-connection-pooling`.
+
+Get PGBouncer URL
+------------------
+
+PGBouncer URL can be extracted via Aiven Command Line interface with the following command replacing the ``INSTANCE_NAME`` parameter with the name of your instance::
+
+    avn service get INSTANCE_NAME --json | jq -r '.connection_info.pgbouncer'
+
+The output will be similar to the below::
+
+    postgres://avnadmin:xxxxxxxxxxx@demo-pg-dev-advocates.aivencloud.com:13040/pgbouncer?sslmode=require
+
+Connect to PGBouncer
```

Comment:
I think that the user knows how to use `psql` let's just say "connect" and go ahead and show the command to run.

Reply:
removed the extra step